lib/3scale/dotenv.rb

```diff
@@ -0,0 +1,17 @@
+# Ensure the proper .env file is loaded
+#
+# Don't load any .env file for production
+# First, try to load `.env.#{ENV['RACK_ENV']}`
+# If doesn't exist, try to load .env.test
+# If doesn't exist, try to load .env
+begin
```

Comment:
Hm... I think I already posted this comment, but I can't find it anywhere :sweat_smile:  So here it goes again:

```
def env_file
  file = ".env.#{ENV['RACK_ENV']}"
  return file if File.exists?(file)

  File.exists?('.env.test') ? '.env.test' : '.env'
end

begin
  return if ENV['RACK_ENV'] == 'production'

  require 'dotenv'

  Dotenv.load env_file
end
```

This is to avoid checking the file existence and assignment for he same file name twice. Not necessary to have an extra method, but I thought it was easier.